src/item/views/modals/AddFragmentToCollection.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { FunctionComponent, useState } from 'react';
+
+import {
+	Button,
+	Column,
+	Form,
+	FormGroup,
+	Grid,
+	Modal,
+	ModalBody,
+	ModalFooterLeft,
+	ModalFooterRight,
+	RadioButton,
+	Select,
+	Spacer,
+	TextInput,
+} from '@viaa/avo2-components';
+import { Avo } from '@viaa/avo2-types';
+import './AddFragmentToCollection.scss';
+
+interface AddFragmentToCollectionProps {
+	externalId: string;
+	itemInfo: Avo.Item.Response;
+	isOpen: boolean;
+	onClose: () => void;
+}
+
+export const AddFragmentToCollection: FunctionComponent<AddFragmentToCollectionProps> = ({
+	externalId,
+	itemInfo,
+	isOpen,
+	onClose = () => {},
+}: AddFragmentToCollectionProps) => {
+	// TODO create endpoint for existing collections + call and put in store
+	const [existingCollections, setExistingCollections] = useState([]);
+	const [createNewCollection, setCreateNewCollection] = useState(false);
+	const [selectedCollection, setSelectedCollection] = useState('');
+
+	const addItemToCollection = () => {
+		onClose();
+	};
+
+	return (
+		<Modal
+			title="Voeg fragment toe aan collectie"
+			size="auto"
+			scrollable={true}
+			isOpen={isOpen}
+			onClose={onClose}
+		>
+			<ModalBody>
+				<div id="addFragmentToCollection">
+					<Spacer>
+						<Form>
+							<Grid>
+								<Column size="2-7">
+									<video
+										src={`${itemInfo.thumbnail_path.split('/keyframes')[0]}/browse.mp4`}
+										placeholder={itemInfo.thumbnail_path}
+										style={{ width: '100%', display: 'block' }}
+										controls={true}
+									/>
+								</Column>
+								<Column size="2-5">
+									<FormGroup label="Title">
+										<TextInput id="title" value={itemInfo.dc_title} />
```

Comment:
TS error `Property 'dc_title' does not exist on type 'ItemResponse'`

Reply:
this is already fixed in the other PR: https://github.com/viaacode/avo2-client/pull/40

Reply:
Alright, I've approved PR #40